src/app/package-json.js

```diff
@@ -107,7 +108,10 @@ const packageJSON = (current, context) => {
       ]),
 
       'build:js:browserify': scriptify([
-        'browserify . -s %s -o dist/%s.js'
+        'browserify .',
+        '-t babelify',
+        '-s ' + context.functionName,
```

Comment:
I was about to ask this very question. I'm not sure how `functionName` "makes sense" and `pluginName` doesn't...